src/ziggurat/metrics.clj

```diff
@@ -35,9 +37,11 @@
 
 (defstate statsd-reporter
   :start (do (log/info "Initializing Metrics")
+             (prometheus-exporter/start-http-server)
              (initialise-metrics-library)
```

Comment:
Write code to check if prometheus is enabled. If enabled do not use statsd. Use prometheus instead